.github/pull_request_template.md

```diff
@@ -1,20 +1,15 @@
 # Description
 
-Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
-
-
-## Type of change
-
-Please delete options that are not relevant.
-
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] This change requires a documentation update
-
-# Checklist:
-
-- [ ] Backwards compatible
-- [ ] My code follows the style guidelines of this project
-- [ ] I have performed a self-review of my code
-- [ ] I have made corresponding changes to the documentation (If applicable)
+Please include a summary of the changes and the related issue to help is review the PR better and faster.
+
+# Checklist for adding new integration:
+
+- [ ] Defined `APIS` in constants [folder](../src/constants/instrumentation/).
+- [ ] Updated `SERVICE_PROVIDERS` in [common.ts](../src/constants/common.ts)
+- [ ] Created a folder under [instrumentation](../src/instrumentation/) with the name of the integration with atleast `patch.ts` and `index.ts` files.
+- [ ] Enabled instrumentation in [init.ts](../src/init/init.ts) & [types.ts](../src/init/types.ts) files.
```

Comment:
Reword to added instrumentation in `allInstrumentations` constant in `init.ts` and to the `InstrumentationType` in `types.ts`

Reply:
done